android/gltfio-android/src/main/java/com/google/android/filament/gltfio/AssetLoader.java

```diff
@@ -123,7 +141,7 @@ public void enableDiagnostics(boolean enable) {
     /**
      * Frees all memory associated with the given {@link FilamentAsset}.
      */
-    public void destroyAsset(@Nullable FilamentAsset asset) {
+    public void destroyAsset(@NonNull FilamentAsset asset) {
```

Comment:
Not sure if it was intended to be `@Nullable` but the current implementation would throw an NPE if it was null so it seems like it was unintentional.

Reply:
Thanks for catching this.